data/json/mutations/mutation_spells.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "id": "cephalopod_ink_spray",
+    "type": "SPELL",
+    "name": "Ink spray",
+    "description": "Blind enemies for a short time with an ink gland discharge.",
+    "effect": "cone_attack",
+    "effect_str": "blind",
+    "affected_body_parts": [ "EYES" ],
+    "valid_targets": [ "hostile" ],
+    "max_level": 5,
+    "min_damage": 0,
+    "max_damage": 1,
+    "damage_increment": 0.0,
+    "min_aoe": 0,
+    "max_aoe": 0,
+    "aoe_increment": 0.2,
+    "min_range": 0,
+    "max_range": 3,
+    "range_increment": 0.0,
```

Comment:
with a range increment of 0, your range will always be 0 (which means it always targets the caster) even with RANDOM_TARGET because that goes up to your max range in a random directoin.

Reply:
I finally figured it out, if i set min_range 0 my range will be 0 at lvl 0 :)